src/Windowing/Silk.NET.Windowing.Common/Structs/WindowOptions.cs

```diff
@@ -120,26 +121,63 @@ public WindowOptions
             VSync = isVSync;
         }
 
+        private static Lazy<WindowOptions> _default = new Lazy<WindowOptions>
```

Comment:
Why lazies? This is probably more memory noise than just the WindowOptions.

Reply:
GetEntryAssembly & friends throw in CoreRT, so the static constructor of WindowOptions throws if this is not lazy.

Reply:
Should probably prefer try-catching the assembly name assignment then, as this isn't the best solution.